src/libraries/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.cs

```diff
@@ -12,126 +12,220 @@ public partial interface INormalizeForIsolatedStorage
     }
     public abstract partial class IsolatedStorage : System.MarshalByRefObject
     {
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
```

Comment:
you can apply the attributes on the class/assembly level since it looks like the whole class is unsupported.

Reply:
Done!